### ISSUE TITLE
Caret always on last word

### ISSUE THREAD

**Issue by dhinugraha** [furcan/KioskBoard#19]
Hi,
Let say I have initial value in input text : 'ABC DEF'
then I want to insert 123 between C and D. 
It must to be : 'ABC123 DEF'
But when I type 123, VK always produce : 'ABC1 DEF23'

config : 
```
theme: 'material',
capsLockActive: true,
allowRealKeyboard: true,
cssAnimations: true,
cssAnimationsDuration: 360,
cssAnimationsStyle: 'slide',
keysAllowSpacebar: true,
keysSpacebarText: 'Space',
keysFontFamily: 'sans-serif',
keysFontSize: '22px',
keysFontWeight: 'normal',
keysIconSize: '25px',
allowMobileKeyboard: true,
autoScroll: true
```

**Comment by furcan** [comment]
Hi @dhinugraha 

Unfortunately, you have to click the exact selection index that you wanted for each time on the inputs. Otherwise, the next value that you have pressed will be added to the end automatically. 

I couldn't find a proper solution for this situation that will be worked with all browsers. (Any help is welcomed.)

Thanks.
Furkan.

**Comment by furcan** [comment]
Hi @dhinugraha 

I created a solution and I will implement it asap.

Thanks,
Furkan.

**Comment by furcan** [comment]
Hi @dhinugraha 

The issue has been fixed (v1.4.0)

Thanks,
Furkan.
